src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -1,73 +1,60 @@
-              </button>
-            </span>
+<div class="login-pf-page">
+  <div class="branding-border"></div>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-6 col-lg-offset-3">
+        <header class="login-pf-page-header">
+          <img class="login-pf-brand" src="assets/branding/RHCSD-Login-Logo.svg" alt=" logo" />
+        </header>
+        <div class="row">
+          <div class="col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
+            <div class="card-pf">
+              <header class="login-pf-header">
+                <select class="selectpicker">
+                  <option>English</option>
+                </select>
+                <h1>Log In to Your Account</h1>
+              </header>
+              <form name="loginForm" (ngSubmit)="login()" #loginForm="ngForm" novalidate>
+                <div class="form-group form-group has-feedback" [ngClass]="{'has-error': (loginForm.submitted || username.dirty) && username.invalid}">
+                  <label class="sr-only" for="email">Email address</label>
+                  <input name="username" [(ngModel)]="model.username" #username="ngModel" type="text" placeholder="Enter your username" class="form-control" required autofocus>
+                  <div class="help-block" *ngIf="(loginForm.submitted || username.dirty) && username.invalid">Username is required</div>
+                </div>
+                <div class="form-group" class="form-group has-feedback" [ngClass]="{'has-error': (loginForm.submitted || password.dirty) && password.invalid}">
+                  <label class="sr-only" for="password">Password
+                  </label>
+                    <input id="password" name="password" [(ngModel)]="model.password" #password="ngModel" type="password" placeholder="Enter your password" class="form-control" required>
+                  <div class="help-block" *ngIf="(loginForm.submitted || password.dirty) && password.invalid">Password is required
+                  </div>
+                </div>
+                <div class="form-group login-pf-settings">
+                  <label class="checkbox-label">
+                    <input type="checkbox" id="stay_signed_in" name="stay_signed_in" [(ngModel)]="model.stay_signed_in"> Keep me logged in for 30 days
+                  </label>
+                </div>
+                <button type="submit" class="btn btn-primary btn-block btn-lg" value="Login">Login</button>
+              </form>
+            </div>
+            <div class="login-logo">
+              <img src="/assets/branding/Logo_RH_RGB_Reverse.png" class="redhat_logo" alt= "Redhat">
+            </div>
+            <footer class="login-pf-page-footer">
+              <ul class="login-pf-page-footer-links list-unstyled">
+                <li><a class="login-pf-page-footer-link" href="#">Terms of Use</a></li>
```

Comment:
@julienlim any idea on how to create these contents? Or is it better to remove them? At least the legally binding contents (`Terms of Use` and `Privacy Policy` should be provided by the legal team).

Should we understand `Help` as a general help (RHCS docs) or login page specific?